addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComLighting1Message.java

```diff
@@ -175,6 +199,55 @@ public String getDeviceId() {
     }
 
     @Override
+    public Command convertToCommand(RFXComValueSelector valueSelector) throws RFXComException {
+
+        Command OHcommand;
+
+        if (valueSelector.getItemClass() == SwitchItem.class) {
+
```

Comment:
Changed the OHcommand to ohCommand. The whitespace follows the style of the existing code, all of which will be tidied up by the simplification PR.